rest/blip_sync.go

```diff
@@ -202,6 +203,12 @@ func (ctx *blipSyncContext) close() {
 		ctx.db.DatabaseContext.DbStats.StatsCblReplicationPull().Add(stat, -1)
 	}
 	close(ctx.terminator)
+
+	waitDuration := time.Millisecond * 100
+	time.Sleep(waitDuration)
+	if count := atomic.LoadInt32(&ctx.activeGoroutines); count > 0 {
+		ctx.Logf(base.LevelWarn, base.KeyAll, "BLIP context closed after %s with %d orphaned goroutines!", waitDuration, count)
```

Comment:
Switch to expvar, as it will be more useful knowing the full count without parsing the logs, and knowing the context doesn't provide that much additional debug value.  Ideally we shouldn't initialize this stat to zero, and only write it if it happens.